cluster.c

```diff
@@ -621,20 +621,16 @@ void ShardingInfoRDBSave(RedisModuleIO *rdb)
         return;
     }
 
-    /* When saving, skip shardgroup #1 which is the local cluster */
-    RedisModule_SaveUnsigned(rdb, si->shard_groups_num - 1);
```

Comment:
we know serialize out the local shardgroup as well as it can change and if if its snapshotted, we wont have a log entry for it.